### ISSUE TITLE
Export Board -> File not found (unable to export one out of 10 boards)

### ISSUE THREAD

**Issue by GYKADSC** [wekan/wekan#1796]
## Export Board results in the following error:

File not found
Firefox can’t find the file at http://xxx.xxx.xxx.xx:9000/api/boards/333fnw5z988WsTkvB/export?authToken=_yDgc1wgIN0nFx95JcNN-PALGirlJVZOgW4XglPRXwm.

**Server Setup Information**:

* Wekan 1.02
* Runs on Docker container
* Forked from wekan/wekan
* MongoDB Version: 3.2.8

**Problem description**:
We have about 10 boards in Wekan and I can export all the boards except one single board. But even this board has been exported (backed up) many times since April 2018. Suddenly the export no longer works while all other boards can be exported.

I have no access to the MongoDB for mongodump due to a missing private key and I don't dare to shut down the instance in order to create a new public/private key pair before I have exported all the boards in case of any issue. That is why I wanted to export all the boards but it does not work for that specific board.

**Comment by xet7** [comment]
@GYKADSC 

I presume this is on AWS. 

1) Create snapshot of EC2 instance

2) Attach snapshotted EBS volume to other EC2 instance

3) Copy raw MongoDB files of Wekan to new server

4), When MongoDB is not running on new server, replace MongoDB raw database files with those you copied from old server

5) Start Wekan

6) If export does not work, try to upgrade to newest Wekan, and try export again.

If you need any help with this, you can use my Commercial Support https://wekan.team

**Comment by d3dbit** [comment]
Hello @GYKADSC , @xet7 ,

I had a similar issue. 
I couldn't export one of my all boards. 
My workaround: [#1740](https://github.com/wekan/wekan/issues/1740)

**Comment by GYKADSC** [comment]
@xet7, It is not on AWS but on Oracle Cloud Container Classic. I have now followed the workaround given by @d3dbit and copied the cards over to a new board. Took some time but works! I have set up a stack with automated mongodump and will migrate to that new environment to avoid that issue in the future. Also I am not sure about the root cause? I there something like a handling error which caused the board to become "un-exportable"?
